### PR TITLE
rofi: 1.7.5 -> 1.7.6

### PR DESCRIPTION
**Pull request by Mange**
New dependency: xcbutilkeysyms

Despite the [1.7.6 release](https://github.com/davatorium/rofi/releases/tag/1.7.6) being marked as a patch release, it contains a lot of new features, some dependency updates, a breaking plugin ABI update, and several years of other misc changes. Binary plugins built for 1.7.5 will not load in 1.7.6 and vice versa.

I'm not sure if we should treat this similarly to a minor/major version and add release notes or not. 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Mange** [comment]
RE: `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`

It's taking its sweet time and I get some build failures, probably because it relies on Nvidia graphics cards to build some CUDA drivers. I'm not sure why it wants to build VMs and the game Planetary Annihalation in order to verify this.

**Comment by lucasew** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 369121`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>rofi-file-browser</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>clerk</li>
    <li>i3-layout-manager</li>
    <li>pinentry-rofi</li>
    <li>plasma-hud</li>
    <li>rofi</li>
    <li>rofi-blezz</li>
    <li>rofi-calc</li>
    <li>rofi-emoji</li>
    <li>rofi-emoji-wayland</li>
    <li>rofi-games</li>
    <li>rofi-menugen</li>
    <li>rofi-pass</li>
    <li>rofi-pass-wayland</li>
    <li>rofi-pulse-select</li>
    <li>rofi-screenshot</li>
    <li>rofi-systemd</li>
    <li>rofi-top</li>
    <li>rofi-unwrapped</li>
    <li>rofi-vpn</li>
    <li>rofi-wayland</li>
    <li>rofi-wayland-unwrapped</li>
    <li>rofimoji</li>
    <li>rofimoji.dist</li>
    <li>todofi-sh</li>
    <li>wl-mirror</li>
  </ul>
</details>

**Review by lucasew — APPROVED** [review]
That failing package was failing before already

**Comment by bew** [comment]
Thanks for the update PR!
I also noticed the inconsistency about the patch release having so many changes.

Yes please add a line in release note


>> ❌ 1 package failed to build:
>>    rofi-file-browser
>
> That failing package was failing before already

 Do you know what's going wrong with it?

_I don't have access to my computer for a couple days still_

**Comment by thiagokokada** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 369121`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>rofi-file-browser</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>clerk</li>
    <li>i3-layout-manager</li>
    <li>pinentry-rofi</li>
    <li>plasma-hud</li>
    <li>rofi</li>
    <li>rofi-blezz</li>
    <li>rofi-calc</li>
    <li>rofi-emoji</li>
    <li>rofi-emoji-wayland</li>
    <li>rofi-games</li>
    <li>rofi-menugen</li>
    <li>rofi-pass</li>
    <li>rofi-pass-wayland</li>
    <li>rofi-pulse-select</li>
    <li>rofi-screenshot</li>
    <li>rofi-systemd</li>
    <li>rofi-top</li>
    <li>rofi-unwrapped</li>
    <li>rofi-vpn</li>
    <li>rofi-wayland</li>
    <li>rofi-wayland-unwrapped</li>
    <li>rofimoji</li>
    <li>rofimoji.dist</li>
    <li>todofi-sh</li>
    <li>wl-mirror</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]

**Comment by Mange** [comment]
Added a release note and squashed into the same commit.

File: nixos/doc/manual/release-notes/rl-2505.section.md
Comment (thiagokokada) [inline nixos/doc/manual/release-notes/rl-2505.section.md:1]: Can you please don't mess with other release note entries? It would be fine to be done in a separate PR, but mixing those fixes all in one PR makes it difficult to review.

Comment (Mange) [reply nixos/doc/manual/release-notes/rl-2505.section.md:1]: I apologize. I did not realize my autoformatter ran. I'll fix that! :bow:

**Comment by thiagokokada** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 369121`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
  </ul>
</details>
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>rofi-file-browser</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 25 packages built:</summary>
  <ul>
    <li>clerk</li>
    <li>i3-layout-manager</li>
    <li>pinentry-rofi</li>
    <li>plasma-hud</li>
    <li>rofi</li>
    <li>rofi-blezz</li>
    <li>rofi-calc</li>
    <li>rofi-emoji</li>
    <li>rofi-emoji-wayland</li>
    <li>rofi-games</li>
    <li>rofi-menugen</li>
    <li>rofi-pass</li>
    <li>rofi-pass-wayland</li>
    <li>rofi-pulse-select</li>
    <li>rofi-screenshot</li>
    <li>rofi-systemd</li>
    <li>rofi-top</li>
    <li>rofi-unwrapped</li>
    <li>rofi-vpn</li>
    <li>rofi-wayland</li>
    <li>rofi-wayland-unwrapped</li>
    <li>rofimoji</li>
    <li>rofimoji.dist</li>
    <li>todofi-sh</li>
    <li>wl-mirror</li>
  </ul>
</details>

**Review by thiagokokada — APPROVED** [review]
